### PR TITLE
[GLT-3758] add missing requisition links

### PR DESCRIPTION
**Pull request by mengjenn**
Added missing requisition links to the informatics reviews, draft reports, and final reports table. As per [GLT-3758](https://jira.oicr.on.ca/browse/GLT-3758)

### CODE REVIEW

File: ts/data/requisition.ts [hunk 1]
```diff
@@ -1,4 +1,11 @@
-import { CellStatus, makeIcon, makeNameDiv } from "../util/html-utils";
+// Informatics Review Table
```
Comment (djcooke) [inline ts/data/requisition.ts:1]: This comment might be misleading here as the file includes table definitions for 2 other tables as well

File: ts/data/requisition.ts [hunk 2]
```diff
@@ -49,12 +56,14 @@ function qcStatusColumn(
...
+  sortType: "text",
   addParentContents(requisition, fragment) {
-    fragment.appendChild(
-      makeNameDiv(requisition.name, urls.miso.requisition(requisition.id))
-    );
+    const nameDiv = document.createElement("div");
+    nameDiv.className = "flex flex-row space-x-2 items-center";
+    addLink(nameDiv, requisition.name, urls.dimsum.requisition(requisition.id));
+    addMisoIcon(nameDiv, urls.miso.requisition(requisition.id));
+    fragment.appendChild(nameDiv);
```
Comment (djcooke) [inline ts/data/requisition.ts:65]: `makeNameDiv` can do all of this - there's an optional `dimsumUrl` parameter

File: ts/data/requisition.ts (hunk 1 above)
Comment (mengjenn) [reply ts/data/requisition.ts:1]: Removed

**Review by djcooke — APPROVED** [review]

**Comment by djcooke** [comment]
Please include the ticket number in commit message when you squash

**Pull request merged**